app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
+  HELPFUL = {
+    'No' => -1,
+    'Kind of' => 0,
+    'Yes' => 1
+  }.freeze
+
   belongs_to :commentable, polymorphic: true, touch: true
 
```

Comment:
Be good to add some validations (and unit tests) for the `#helpful` attribute, i.e. to make sure values are any of `[-1, 0, 1]`.